posts/2024-05-10_new_admiral_ex.../new_admiral_extension_packages_admiralpeds_coming_soon.qmd

```diff
@@ -0,0 +1,70 @@
+author:
+  - name: Fanny Gautier
+  - name: Lina Patil
+description: "We are very excited to announce that new extension packages are expected very soon in the {admiral} family: {admiralpeds}, a pediatrics extension, and another extension (name TBC) for obesity in clinical trials"
+date: "2024-05-10"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [Metadata, ADaM, Community]
+# feel free to change the image
+image: "coming_soon.png"
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2024-05-10_new_admiral_ex..."
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+[![](admiralpeds.png){fig-align="center" width="220"}](https://pharmaverse.github.io/admiralpeds/index.html)
+
+📢 In the pharmaverse, we are not just adopting new tools, but also fostering a culture of collaboration, innovation and a commitment to continuous learning. We are thus thrilled to update you on the latest developments from our {admiral} family team: [`{admiralpeds} 0.1`](https://pharmaverse.github.io/admiralpeds/index.html) is __coming soon__ [on CRAN](https://cran.r-project.org)!
+
+`{admiralpeds}` is a pediatrics extension package for  `{admiral}`. Its first release is planned by July 2024.
+
+# Collaboration
+
+7 companies (Roche, Pfizer, Amgen, BMS, J&J, Novartis, and Cytel) and David Freedman, a passionate consultant retired from CDC, are working together to create this extension 🤝.
+
+# Purpose
+
+The purpose of this new extension is to provide a complementary (to `{admiral}`) toolbox that enables users to develop specific ADaMs datasets for pediatric clinical trials.
+Initially the package will focus on child growth/development charts: [WHO and CDC growth charts](https://www.cdc.gov/growthcharts/who_charts.htm), among infants and children ages birth to 19 years included. 
+
+For guidance and a smooth use of this new extension and as for the main [`{admiral}`](https://pharmaverse.github.io/admiral/index.html) package, ADaMs datasets templates, test data, [website](https://pharmaverse.github.io/admiralpeds/index.html) and vignettes gonna be available at the release time.
+
+# New Features
+
+We are starting with 2 new functions: 
+
+* `admiralpeds::derive_params_growth_age()`
+* `admiralpeds::derive_params_growth_height()`
+
+The first function, `derive_params_growth_age()` is based on the current age of the patients, while the second one, `derive_params_growth_height()` depends on the height or length of the patients. The two functions are flexible enough to choose your own metadata. This could either be company dependent, or the pre-defined ones based on WHO and CDC charts, as created in our ADaM template: *bmi_for_age, height_for_age, weight_for_age, who_hc_for_age, who_wt_for_ht_lgth*. You can also choose the desired parameters (e.g. "BMI", "HEIGHT", "WEIGHT", "HEAD CIRCUMFERENCE").
```

Comment:
```suggestion
The first function, `derive_params_growth_age()` is based on the current age of the patients, while the second one, `derive_params_growth_height()` depends on the height or body length of the patients. The two functions are flexible enough to choose your own metadata. This could either be company dependent, or the pre-defined ones based on WHO and CDC charts, as created in our `ADVS` ADaM template. You can also choose the desired parameters (e.g. "BMI", "HEIGHT", "WEIGHT", "HEAD CIRCUMFERENCE").
```

Reply:
Edoardo asked to add something about the metadata. Could you please confirm to finally drop it again ?
Thanks.

Reply:
Hi @manciniedoardo, 
Could you please confirm this is fine to add that details in the vignette only ?